Dockerfile

```diff
@@ -72,6 +76,10 @@ ENV BUILD_BRANCH=${BUILD_BRANCH}
 
 RUN make build-go GO_BUILD_TAGS=${GO_BUILD_TAGS} WIRE_TAGS=${WIRE_TAGS}
```

Comment:
How this works?  You run the build-go twice.
does it make architecture aware file structure? otherwise, it should override

Reply:
build-go result path prefix is /tmp/grafana/bin/linux-${TARGETARCH} so first run will produce the linux-amd64 binaries second run with GOARCH=arm64 CC=/arm64/usr/bin/gcc will produce linux-arm64 binaries